src/main/java/com/hazelcast/commandline/HazelcastCommandLine.java

```diff
@@ -0,0 +1,99 @@
+package com.hazelcast.commandline;
+
+import com.hazelcast.commandline.member.MemberCommandLine;
+import picocli.CommandLine;
+
+import java.io.PrintStream;
+import java.util.List;
+import java.util.concurrent.Callable;
+
+import static com.hazelcast.instance.BuildInfoProvider.getBuildInfo;
+import static picocli.CommandLine.*;
+
+@Command(
+        name = "hazelcast",
+        description = "Utility for the Hazelcast IMDG operations." +
+                "%n%n" +
+                "Global options are:%n",
+        versionProvider = HazelcastVersionProvider.class,
+        mixinStandardHelpOptions = true,
+        sortOptions = false
+)
+public class HazelcastCommandLine implements Callable<Void> {
+
+    public static String HAZELCAST_HOME = System.getProperty("user.home") + "/.hazelcast";
+    public final static String SEPARATOR = System.getProperty("file.separator");
+
+    @Mixin(name = "verbosity")
+    private Verbosity verbosity;
+
+    public Void call() {
+        return null;
+    }
+
+    public static void main(String[] args) {
+        runCommandLine(System.out, System.err, true, args);
+    }
+
+    static void runCommandLine(
```

Comment:
1. `private`?
2. The formatting looks weird. Why not just keep parameters in one line?
```
static void runCommandLine(PrintStream out, PrintStream err, boolean shouldExit, String[] args) {
```

Reply:
Changed 👍